lib/fedex.rb

```diff
@@ -52,4 +52,11 @@ module Fedex
   require 'fedex/version'
   #Exceptions: Fedex::RateError
   class RateError < StandardError; end
+
+  # Indicates a general fault in the Fedex web service
+  class TrackingInformationFault < RateError; end
```

Comment:
Can we call this `FedexServerError` and the other error `FedexRequestError`?

`TrackingInformationFault` is too vague, if I saw this and `TrackingInformationError` I would not know the difference without having to dig up this file and look at the comments. I don't think we should depend on people's knowledge of the nuanced difference between `fault` and `error`. The comments are opaque as well, should just use what you wrote in the PR - also, it's a code smell if you have to use a comment to explain what the error means.

Reply:
Done